### PR TITLE
Added starsigns to random agent

### PR DESCRIPTION
**Pull request by nikqo**
Also added some spacing for more readability

### CODE REVIEW

File: dgisim/src/agents.py [hunk 1]
```diff
@@ -115,6 +120,24 @@ def choose_action(self, history: List[GameState], pid: GameState.Pid) -> PlayerA
...
+                    card,
+                    CharacterTargetInstruction(
+                        tmp_dices,
+                        StaticTarget(
+                            pid,
+                            Zone.CHARACTER,
+                            active_character.get_id(),
+                        )
+                    )
+                )
```
Comment (Jarvis-Yu) [inline dgisim/src/agents.py:139]: just tested in Genshin with npc, player cannot play Starsigns when the active character has their energy full. pls add the corresponding condition here. (just nitpick)

**Review by Jarvis-Yu — CHANGES_REQUESTED** [review]
Generally looks good, just a nitpick

File: dgisim/src/agents.py (hunk 1 above)
Comment (nikqo) [reply dgisim/src/agents.py]: fixed in latest push

File: dgisim/src/agents.py [hunk 2]
```diff
@@ -126,17 +135,19 @@ def choose_action(self, history: List[GameState], pid: GameState.Pid) -> PlayerA
                 card = Starsigns
                 tmp_dices = ActualDices({Element.ANY: 2})
                 print(card().name())
```
Comment (Jarvis-Yu) [inline dgisim/src/agents.py]: I'd probably move these 3 lines under the condition below
especially the print

**Review by Jarvis-Yu — APPROVED** [review]
LGTM, just one more thing.
I want the commit messages to be more descriptive about the changes in the commit, by which I mean the first line of the commit. You may include more details in the following lines as you wish, but most of the time people only see the first line, so it needs to be more than "update some file or some class".

**Pull request merged**